src/mender-inventory-mender-configure

```diff
@@ -42,40 +85,64 @@ DBUS_REPLY=$(echo "$DBUS_RAW" | sed -ne '/^ *string ".*" *$/ {s/^ *string "\(.*\
 AUTH_TOKEN="$(echo "$DBUS_REPLY" | sed -ne '1p')"
 SERVER="$(echo "$DBUS_REPLY" | sed -ne '2p')"
 
-if [ -z "$AUTH_TOKEN" ]; then
-    echo "mender-inventory-mender-configure: An authentication token could not be obtained over DBus."
-    # Don't produce an error. Most likely mender-auth is not ready or
-    # the device hasn't been accepted
+report_config $AUTH_TOKEN $SERVER
+
+if [ $? -eq 0 ]; then
     exit 0
 fi
-if [ -z "$SERVER" ]; then
-    echo "mender-inventory-mender-configure: A server address could not be obtained over DBus."
-    exit 1
-fi
 
-# Include the JWT token in the checksum to report again the
-# configuration to the server in case of new authorization after
-# device decommissioning or connection to a different Mender server
-COMPUTED_CHECKSUM="$( (echo "$AUTH_TOKEN"; cat "$CONFIG") | sha256sum -)"
+# here: expired or inexistent token
+
+tmp_file=$(mktemp)
+
+(dbus-monitor \
+    --system \
+    "type='signal',interface='io.mender.Authentication1'" \
+    | sed -n \
+    '/JwtTokenStateChange/{n;s/.*string "\(.*\)"/\1/p;n;s/.*string "\(.*\)"/\1/p;q}' \
```

Comment:
This sed command looks fragile and difficult to maintain to me. Why do we want that? Isn't it good enough just knowing if it was _any_ signal during the timeout time?